### PR TITLE
Ev/default fields

### PR DESCRIPTION
**Pull request by vilsbole**
- connect Address and Amount to form
- include validation
- restyle slider
- minor refactors

## Quality Assurance

- [x] The branch name is in lowercase-kebab-case with no prefix (unless it was created from Clubhouse)
- [x] The base branch is develop or gau (no nested branches)
- [x] This is related to a maximum of one Clubhouse story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [x] If code is copied from existing directories, there is an explanation of why this is necesary in the description/changes, and all copying is done in separate commits to make them easy to filter out

### CODE REVIEW

File: common/v2/types/symbols.ts [hunk 1]
```diff
@@ -1,4 +1,1317 @@
-// @TODO:SEND add all token symbols to type
-type TSymbol = 'ETH' | 'USD' | 'generic';
+// Create union type from array
+// https://stackoverflow.com/a/45257357/2057532
+// Solution is valid until we update to typescript 3.4 where we can then use
+// const symbol = [] as const
+export type Lit = string | number | boolean | undefined | null | void | {};
+export const tuple = <T extends Lit[]>(...args: T) => args;
+
+const symbols = tuple(
```
Comment (blurpesec) [inline common/v2/types/symbols.ts]: This won't work as the assets / their symbols change any time we update tokens.

Comment (vilsbole) [reply common/v2/types/symbols.ts]: Wouldn't be more like incrementing a list with the new tokens? Or do the symbols also change?

This was just a suggestion to give a bit more confidence when using a symbol|ticker to express the idea that it's more than a simple string and that we rely on it for filtering. I pushed changes that use create the equivalent of a type alias using enum + intersection with the base structure (aka. string). With this we have one way equality checks where a TSymbol can be assigned to a string, but a string needs to be cast as TSymbol.

Comment (blurpesec) [reply common/v2/types/symbols.ts]: > Wouldn't be more like incrementing a list with the new tokens? Or do the symbols also change?

Yeah currently everything about the tokens config can change from pulling tokens down from ethereum-lists/tokens.

Also adding custom tokens (tokens that arent specified in the list we import) would need to be handled by that type.

Comment (vilsbole) [reply common/v2/types/symbols.ts]: I updated the implementation to this: https://github.com/MyCryptoHQ/MyCrypto/blob/7387450e14ab41a95238d6c950e371eaf39a95b0/common/v2/types/symbols.ts
Let me know what you think.

Comment (blurpesec) [reply common/v2/types/symbols.ts]: Looks good!

**Pull request merged**